packages/blog/src/pages/post/[postId]/[postName]/index.tsx

```diff
@@ -84,20 +84,20 @@ export const getServerSideProps: GetServerSideProps<PostPageProps | WithErrorPro
     return handlePageError(error as Error, res)
   }
 
-  let relatedPosts: Post[] | undefined = undefined
+  let sameCategoryPosts: Post[] | undefined = undefined
   if (post?.categories?.length) {
-    const relatedPostsRequest = getLatestPosts({
+    const sameCategoryPostsRequest = getLatestPosts({
       locale: locale as AppLocales,
       category: post?.categories?.[0]?.code,
     })
-    const [responseRelatedPost] = await Promise.all([relatedPostsRequest])
-    relatedPosts = responseRelatedPost
+    const [responsesameCategoryPost] = await Promise.all([sameCategoryPostsRequest])
+    sameCategoryPosts = responsesameCategoryPost?.filter(sameCategoryPost => sameCategoryPost.id !== post?.id)
```

Comment:
what is the limit for the number of posts shown?

Reply:
no limit at the moment! Adding one. I think 4 should be enough